modules/highlights.js

```diff
@@ -0,0 +1,31 @@
+const Augur = require('augurbot');
+const Module = new Augur.Module();
+Module.addCommand({name: "fetchhighlights", 
+    permissions: (msg) => ['337713155801350146'].includes(msg.author.id),
+    process: async(msg, args) =>{
+        let date = new Date()
+        let after = new Date(date.getFullYear(), date.getMonth(), 1)
+        let channel = msg.guild.channels.cache.get('737557773000245298') //#highlight-submissions
+        async function messageFetcher(limit = 500) {
```

Comment:
Why is this defined as a function here when it's just called once immediately after?